docs/data-modeling/model.mdx

```diff
@@ -5,519 +5,208 @@ description: "Models allow you to define the structure of your data and how it c
+```bash title="Introspect your data source:"
 ddn connector introspect <connector_name>
 ```
 
-```bash title="Then, add your model:"
-ddn model add <connector_name> users
-```
+Whenever you update your data source, you can run the above command to fetch the latest resources.
 
-  </TabItem>
-  <TabItem value="ClickHouse" label="ClickHouse">
-
-**Via a new table or view**
-
-```sql title="Create a new table or view in your ClickHouse database:"
-CREATE TABLE users (
-    id UInt32,
-    name String
-)
-ENGINE = MergeTree()
-ORDER BY id;
-
-INSERT INTO users (id, name) VALUES
-    (1, 'Alice Johnson'),
-    (2, 'Bob Smith'),
-    (3, 'Charlie Davis');
+```bash title="Show the resources discovered from your data source:"
+ddn connector show-resources <connector_name>
 ```
 
-```bash title="Use the DDN CLI to introspect your ClickHouse instance:"
-ddn connector introspect <connector_name>
-```
+This is an optional step and will output a list of resources that DDN discovered in your data source in the previous
+step.
 
-```bash title="Then, add your model:"
-ddn model add <connector_name> users
+```bash
+ddn model add <connector_link_name> <collection_name>
 ```
 
-**Via an existing table or view**
-
-If you already have a table present, you can add it by introspecting your data source and referencing it by name.
+Or you can optionally add all the models by specifying `"*"`.
 
-```bash title="Use the DDN CLI to introspect your ClickHouse instance:"
-ddn connector introspect <connector_name>
-```
-
-```bash title="Then, add your model:"
-ddn model add <connector_name> users
+```bash
+ddn model add <connector_link_name> "*"
 ```
 
-  </TabItem>
-</Tabs>
+This will add models with their accompanying metadata definitions to your metadata.
 
-```bash title="Create a new build:"
-ddn supergraph build local
-```
+You can now build your supergraph API, serve it, and query your data.
 
-```bash title="Start your services:"
-ddn run docker-start
-```
+:::info Context for CLI commands
 
-```bash title="Open your development console:"
-ddn console --local
-```
+Note that the above CLI commands work without also adding the relevant subgraph to the command with the `--subgraph`
+flag because this has been set in the CLI context. You can learn more about creating and switching contexts in the
+[CLI context](/) section. {/* TODO: Add link */}
 
-```graphql title="You can now query your table:"
-query {
-  users {
-    id
-    name
-  }
-}
-```
+:::
 
-```json title="With a response like this:"
-{
-  "data": {
-    "users": [
-      {
-        "id": 1,
-        "screamedName": "Alice Johnson"
-      },
-      {
-        "id": 2,
-        "screamedName": "Bob Smith"
-      },
-      {
-        "id": 3,
-        "screamedName": "Charlie Davis"
-      }
-    ]
-  }
-}
-```
+For a walkthrough on how to create a model, see the [Model Tutorials](#model-tutorials) section.
 
-### Extending a model
+### Via native query
```

Comment:
Since we're calling out a new subsection here for organization, would it be easier for a user to see there are multiple ways of creating a model by adding an h3 at the start of the `Create a model` section? We could call it something like `From a source entity` or something along those lines to differentiate.